<launch>

	<arg name="graph_file" default="$(find lab4)/gates_dense_obstacles_mppi_final.graphml"/>
  <arg name="visualize" default="false"/>
  <node pkg="lab4" type="PlannerNode.py" name="planner_node" output="screen">
    <param name="graph_file" value="$(arg graph_file)"/>
    <param name="visualize" type="bool" value="$(arg visualize)"/>
  </node>

</launch>
